libraries/plugins/blockchain_history/devcommittee_history_api.cpp

```diff
@@ -0,0 +1,155 @@
+#include <scorum/blockchain_history/devcommittee_history_api.hpp>
+#include <scorum/blockchain_history/blockchain_history_plugin.hpp>
+#include <scorum/blockchain_history/schema/history_object.hpp>
+#include <scorum/app/api_context.hpp>
+#include <scorum/app/application.hpp>
+#include <scorum/blockchain_history/schema/operation_objects.hpp>
+#include <scorum/common_api/config.hpp>
+#include <scorum/protocol/operations.hpp>
+
+#include <map>
+
+namespace scorum {
+namespace blockchain_history {
+
+namespace detail {
+class devcommittee_history_api_impl
+{
+public:
+    scorum::app::application& _app;
+
+public:
+    devcommittee_history_api_impl(scorum::app::application& app)
+        : _app(app)
+    {
+    }
+
+    template <typename history_object_type, typename fill_result_functor>
+    void get_history(uint64_t from, uint32_t limit, fill_result_functor& funct) const
+    {
+        const auto db = _app.chain_database();
+
+        FC_ASSERT(limit > 0, "Limit must be greater than zero");
+        FC_ASSERT(limit <= MAX_BLOCKCHAIN_HISTORY_DEPTH, "Limit of ${l} is greater than maxmimum allowed ${2} ",
+                  ("l", limit)("2", MAX_BLOCKCHAIN_HISTORY_DEPTH));
+
```

Comment:
NOTE:
Old interface checked FC_ASSERT(from >= limit, "From must be greater than limit");